PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalInternalClient.kt

```diff
@@ -247,16 +252,25 @@ internal class PayPalInternalClient(
 
     private fun getClientMetadataId(
         context: Context,
+        correlationId: String?,
         callback: (String?) -> Unit
     ) {
         braintreeClient.getConfiguration { configuration, error ->
             if (error != null) {
                 callback(error("No Client Metadata Id"))
             } else {
-                val clientMetadataId = dataCollector.getClientMetadataId(
-                    context,
-                    configuration,
+                val request = DataCollectorRequest(false, correlationId)
```

Comment:
I'm thinking we probably want to add the `hasUserLocationConsent` param to the request object and pass it through here. If we always just pass false, Magnes is limited in the data it collects. I'm not sure if it's needed in this case though